src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6355,28 +6363,35 @@ namespace SimAirServingZones {
 				CalcSysSizing( AirLoopNum ).InpDesHeatAirFlow = CalcSysSizing( AirLoopNum ).DesHeatVolFlow * FractionOfAutosize;
 				FinalSysSizing( AirLoopNum ).InpDesHeatAirFlow = FinalSysSizing( AirLoopNum ).DesHeatVolFlow * FractionOfAutosize;
 			} else if ( SELECT_CASE_var == FlowPerHeatingCapacity ) {
-				if ( FinalSysSizing( AirLoopNum ).HeatingCapMethod == FractionOfAutosizedHeatingCapacity ) {
-					FractionOfAutosize = FinalSysSizing( AirLoopNum ).ScaledHeatingCapacity;
-				}
-				if ( FinalSysSizing( AirLoopNum ).HeatOAOption == MinOA ) {
-					if ( FinalSysSizing( AirLoopNum ).DesHeatVolFlow > 0.0 ) {
-						OutAirFrac = FinalSysSizing( AirLoopNum ).DesOutAirVolFlow / FinalSysSizing( AirLoopNum ).DesHeatVolFlow;
+				if ( FinalSysSizing( AirLoopNum ).HeatingCapMethod == FractionOfAutosizedHeatingCapacity || FinalSysSizing( AirLoopNum ).ScaledHeatingCapacity == DataSizing::AutoSize ) {
```

Comment:
This is confusing. There are 2 tests here. One or the other or both could be true. If HeatingCapMethod != FractionOfAutosizedHeatingCapacity and ScaledHeatingCapacity == AutoSize how do you know what to do? What if HeatingCapMethod == CapacityPerFloorArea, these calculations don't look correct?